ui/src/components/SchemaForm/Templates/ArrayFieldTemplate.tsx

```diff
@@ -53,26 +47,22 @@ const ArrayFieldTemplate = <
   return (
     <div>
       <ArrayFieldTitleTemplate
-        idSchema={idSchema}
+        fieldPathId={fieldPathId}
         title={uiOptions.title || title}
         schema={schema}
         uiSchema={uiSchema}
         required={required}
         registry={registry}
       />
       <ArrayFieldDescriptionTemplate
-        idSchema={idSchema}
+        fieldPathId={fieldPathId}
         description={uiOptions.description || schema.description}
         schema={schema}
         uiSchema={uiSchema}
         registry={registry}
       />
-      <div key={`array-item-list-${idSchema.$id}`}>
-        {items?.map(
-          ({ key, ...itemProps }: ArrayFieldTemplateItemType<T, S, F>) => (
-            <ArrayFieldItemTemplate key={key} {...itemProps} />
-          ),
-        )}
+      <div key={`array-item-list-${fieldPathId.$id}`}>
+        {items}
```

Comment:
Do you know if we are still using our ArrayFieldItemTemplate under the hood, or are we now using the rjsf default?